@types/memoize-one/index.d.ts

```diff
@@ -0,0 +1,12 @@
+export declare type EqualityFn = (
```

Comment:
Instead of adding our own type overrides, can we just update memoize-one to `6.0.0`? Looks like they added types and tightened it up for TypeScript usage: https://github.com/alexreardon/memoize-one/releases
https://github.com/alexreardon/memoize-one/blob/master/src/memoize-one.ts

Reply:
I'll have to check. At first glance, their types still appear to let through untyped function params without warning which is the whole point of my type overrides here